### PR TITLE
feat: add CRD chart to be later used as a dependency

### PR DESCRIPTION
**Pull request by spring1843**
Fixes # 

**Description**

Add CRD chart that publishes to GHCR

**How was this change tested?**

*

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by FernandoMiguel** [comment]
this is a great idea. 
makes the CRDs updates much nicer than most other current approaches

File: hack/release/common.sh [hunk 1]
```diff
@@ -2,9 +2,12 @@
 set -euo pipefail
 
 config(){
+  GITHUB_ACCOUNT="aws"
   AWS_ACCOUNT_ID="071440425669"
   ECR_GALLERY_NAME="karpenter"
   RELEASE_REPO=${RELEASE_REPO:-public.ecr.aws/${ECR_GALLERY_NAME}/}
+  RELEASE_REPO_GH=${RELEASE_REPO_GH:-ghcr.io/${GITHUB_ACCOUNT}/karpenter}
```
Comment (ellistarn) [inline hack/release/common.sh:9]: optional
```suggestion
  RELEASE_REPO_GH=${RELEASE_REPO_GH:-ghcr.io/aws/karpenter}
```

**Review by ellistarn — APPROVED** [review]

File: hack/release/common.sh (hunk 1 above)
Comment (spring1843) [reply hack/release/common.sh:9]: Oops I missed this, but it's parametrized to make developing against a local fork easier, e.g. all you have to do is modify values in the config function.

**Comment by cest-pas-faux** [comment]
Hello, just asking if you're willing to reconsider this (with all the new features since June btw)
https://github.com/aws/karpenter/pull/1956

**Comment by fernandezcuesta** [comment]
Hi, I'm wondering why not all CRDs are included here (i.e. Provisioner's CRD) as opposed to Karpenter's Helm chart.

**Pull request merged**